autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/coalescer/NoCoalescer.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, Optional, Union
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.preprocessing.tabular_preprocessing.coalescer.base_coalescer import BaseCoalescer
+
+
+class NoCoalescer(BaseCoalescer):
+    """
+    Don't perform NoCoalescer on categorical features
```

Comment:
Why?

Reply:
This is a `NoCoalescer` class. This allows the BO model to enable/disable coalescing. 

The choice object selects between MinorityCoalescer and NoCoalescer depending on what gives better performance.

Reply:
I mean i did not get if you mean `Do not perform NO coalescer` or `Do not perform coalescer`.